readme-template.hbs

```diff
@@ -0,0 +1,4 @@
+# bedrock-stats-storage-redis
+
+# API Reference
+{{>main}}
```

Comment:
are we adding the copyright to docs? we may as well get a default template for docs.

Reply:
We have not been adding copyright notices to docs.  Seems like just another thing to maintain IMO.